modelexpress_client/python/modelexpress/weight_transfer/planner/server.py

```
@@ -0,0 +1,106 @@
+    def build(
+        self,
+        regions: list[ResolvedRegion],
+        table: TrainerTable,
+        plan_key: str,
+    ) -> list[RdmaDescriptor]:
+        if plan_key in self._local_cache:
+            return self._local_cache[plan_key]
+
+        try:
+            descriptors = self._build_via_server(regions, plan_key)
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)